src/components/FeaturePanel/FeaturePanelInDrawer.tsx

```diff
@@ -3,17 +3,26 @@ import { FeaturePanel } from './FeaturePanel';
 import { Drawer } from '../utils/Drawer';
 import {
   DRAWER_PREVIEW_HEIGHT,
+  DRAWER_PREVIEW_PADDING,
   DRAWER_TOP_OFFSET,
 } from '../utils/MobilePageDrawer';
 
 const DRAWER_CLASSNAME = 'featurePanelInDrawer';
 
-export const FeaturePanelInDrawer = () => (
-  <Drawer
-    topOffset={DRAWER_TOP_OFFSET}
-    className={DRAWER_CLASSNAME}
-    collapsedHeight={DRAWER_PREVIEW_HEIGHT}
-  >
-    <FeaturePanel />
-  </Drawer>
-);
+export const FeaturePanelInDrawer = () => {
+  const [height, setHeight] = React.useState(DRAWER_PREVIEW_HEIGHT);
```

Comment:
I would prefer `collapsedHeight` it took me some time to understand which height is which.

Also `useState<number>` could be better.